src/libraries/System.Private.Xml/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,8 @@
+﻿<linker>
+  <assembly fullname="System.Private.Xml">
+    <type fullname="System.Xml.LocalAppContextSwitches">
+      <method signature="System.Boolean get_AllowResolvingUrlsByDefault()" body="stub" value="false"
```

Comment:
File name URLs are still valid [URLs](https://en.wikipedia.org/wiki/URL). 

Should the name rather be `AllowResolvingNonFileSystemUrlsByDefault`?

Reply:
API review decided on `System.Xml.XmlResolver.IsNetworkingEnabledByDefault`